Mathlib/Combinatorics/SimpleGraph/Triangle/Basic.lean

```diff
@@ -28,23 +29,163 @@ This module defines and proves properties about triangles in simple graphs.
 
 ## TODO
 
-* Generalise `farFromTriangleFree` to other graphs, to state and prove the Graph Removal Lemma.
+* Generalise `FarFromTriangleFree` to other graphs, to state and prove the Graph Removal Lemma.
```

Comment:
All the changes in this file have (I think) already been made in #12526 and should be removed from this PR.

Reply:
(clarified in PM)